plugin.json

```diff
@@ -23,6 +23,20 @@
         "help_text": "When false, users cannot attach and create Jira issues in Mattermost. Does not affect Jira webhook notifications. After changing this setting to false, disable, then re-enable this plugin in **System Console > Plugins > Plugin Management** to reset the plugin state for all users. \n \n When true, install this plugin to your Jira instance with '/jira install’ to allow users to create and manage issues across Mattermost channels. See [documentation](https://about.mattermost.com/default-jira-plugin-link-application) to learn more.",
         "default": true
       },
+      {
+        "key": "WebhookMaxProcsPerServer",
+        "display_name": "Maximum # of Webhook Processors Per Server",
+        "type": "text",
+        "help_text": "The number of simultaneous go routines that will process webhooks. This is needed because to process a webhook the Jira plugin may have to request information from the Jira instance (such as permissions), and we may not want to overload Jira. \n \n Changes to this setting will take effect when the plugin is restarted.",
```

Comment:
```suggestion
        "help_text": "The number of simultaneous go routines that will process webhooks. This is needed because to process a webhook the Jira plugin may have to request information from the Jira instance (such as permissions), and we may not want to overload Jira. \n \n Changes to this setting will take effect when the plugin is restarted. Default value is 20",
```